azure-pipelines-build-stage-job.yml

```diff
@@ -35,11 +35,11 @@ jobs:
           mkdir upload
           echo 2.1.$(Build.BuildId)_$(Build.SourceBranchName)_$(COMMIT_HASH)_$(Agent.OS)_Release > MM_VERSION
           cat MM_VERSION
-          cargo build --features native --release --bin mm2
+          cargo build --release --bin mm2
```

Comment:
Could you please add a step to build MM2 for WASM and also run tests in a headless browser? It's fine to perform this step on the Linux agent only.

Reply:
Ok, no problem :+1:

Reply:
I'm not sure how we'd build the wasm library.
For example, currently I'm using the [wasm-pack](https://rustwasm.github.io/docs/wasm-bindgen/examples/without-a-bundler.html) to build the MM2 wasm library:
```bash
wasm-pack build --target web
```
Thereby I added only `Test MM2 WASM` step